R/main.R

```diff
@@ -17,5 +17,17 @@ Options:
 
 main <- function(args = commandArgs(TRUE)) {
   dat <- parse_main(args)
-  api(dat$validate, dat$log_level)$run("0.0.0.0", port = dat$port)
+  api <- api(dat$validate, dat$log_level, configure_queue = TRUE)
+  api$run("0.0.0.0", port = dat$port)
+}
+
+
+main_worker <- function() {
+  queue <- Queue$new(configure_queue = FALSE)
+
+  worker <- rrq::rrq_worker$new(
+    queue$controller$queue_id,
+    con = queue$controller$con
+  )
```

Comment:
```suggestion
  worker <- rrq::rrq_worker$new(
    get_queue_id(),
    con = queue$controller$con
  )
```

or is there some side-effect from constructing `Queue` you're also trying to hit?

Reply:
Right, yeah I guess the worker doesn't need to make a queue - they will ned a redis connection, but should be able to make that here similarly, using `get_redis_host`